configure

```diff
@@ -77,15 +76,16 @@ freshen_chef_repo
 pushd chef_repo
 wd="$(pwd)"
 
+if test \( -n "$CHEF_MAGE_PROFILE" -a test -z "$MAGE_KEY" \) -o \( test -z "$CHEF_MAGE_PROFILE" -a test -n "$MAGE_KEY" \); then 
```

Comment:
```suggestion
if test \( -n "$CHEF_MAGE_PROFILE" -a -z "$MAGE_KEY" \) -o \( -z "$CHEF_MAGE_PROFILE" -a -n "$MAGE_KEY" \); then 
```

I don't recall if these were there in my original suggestion but they are spurious.
This should be the thing which might also work in bash where the other thing doesn't.
If not, just switch to `[[ ]]` and we'll port it when we finish the posix switch.

Reply:
This indeed work. Tested the script with these changes and see that behavior is restored if no MAGE variable is enabled and faults if MAGE is present